README.md

```diff
@@ -1,70 +1,109 @@
-# [Loom SDK](https://loomx.io)
+# [Basechain](https://loomx.io)
 
-Loom application specific side chain software development kit.
+Basechain is an interoperable DPoS blockchain that is live in production, EVM-compatible, audited, and battle-tested.
 
-## Documentation
- 
- 
-[Loom SDK Documentation Site](https://loomx.io/developers/)
 
-## Environment Setup
-
-Requirements
+## Prerequisites
 
+* A running Linux or macOS system.
 * Go 1.9+
-* [Dep](https://github.com/golang/dep)
+  You can enter the following command to check that Go is installed on your system:
 
-On Mac
-```
-brew install dep
-```
-and on Linux
-```
-curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-```
+  ```shell
+  go version
+  ```
 
-### To build loomchain with cleveldb
-On Mac 
-```
-brew install leveldb
-```
-and on Linux
-```
-apt-get install libleveldb-dev libleveldb1v5
-```
+  If Go is installed, the output looks like this:
 
-## Installing
+  ```
+  go version go1.13.8 darwin/amd64
+  ```
 
-[Install SDK](https://loomx.io/developers/docs/en/prereqs.html)
+  If Go is not installed or you're running an older version, you can refer to the [Getting Started](https://golang.org/doc/install) page of the Go documentation for more details about how you can install or upgrade Go.
 
-## Building
-Make sure `GOPATH` is defined and run
+* Dep. For details about installing Dep, see the [Dep](https://github.com/golang/dep) page.
 
-```shell
-LOOM_SRC=$GOPATH/src/github.com/loomnetwork/loomchain
-# clone into gopath
-git clone git@github.com:loomnetwork/loomchain.git $LOOM_SRC
-# install deps
-cd $LOOM_SRC
-make deps
-make
-```
+* (OPTIONAL) LevelDB.
 
-## Running
+  If you're running macOS, you can enter the following command to install LevelDB:
 
-```shell
-# init the blockchain with builtin contracts
-./loom init
-# run the node
-./loom run
-```
+  ```shell
+  brew install leveldb
+  ```
+
+  If you're running Linux, you can enter the following command to install LevelDB:
+
+  ```shell
+  apt-get install libleveldb-dev libleveldb1v5
+  ```
+
+* The `GOPATH` environment variable is defined.
+
+  Use the following command to check that the `GOPATH` environment variable is defined:
```

Comment:
Lets assume the reader knows what a GOPATH is, and skip this hand-holding.

Reply:
@enlight Gotcha.